### PR TITLE
BTFSINFRA-328 fix sharness test basic-commands

### PR DESCRIPTION
**Pull request by richlee33**
initial commit to getting the tests found in the test/sharness to run properly.

### CODE REVIEW

File: test/sharness/Rules.mk [hunk 1]
```diff
@@ -8,7 +8,7 @@ DEPS_$(d) := test/bin/random test/bin/multihash test/bin/pollEndpoint \
 	   test/bin/iptb test/bin/go-sleep test/bin/random-files \
 	   test/bin/go-timeout test/bin/hang-fds test/bin/ma-pipe-unidir \
 	   test/bin/cid-fmt
-DEPS_$(d) += cmd/btfs/ipfs
+DEPS_$(d) += cmd/btfs/btfs
```
Comment (jin-tron) [inline test/sharness/Rules.mk]: Is /cmd/btfs/btfs a file? How the file comes from?
I saw we only have cmd/btfs/ipfs.go.

Comment (richlee33) [reply test/sharness/Rules.mk]: for some reason my install has the btfs executable in the location: go-btfs/cmd/btfs. i also have the ipfs.go file but no btfs.go file.

Comment (jin-tron) [reply test/sharness/Rules.mk]: ok, so it is binary file.

Comment (richlee33) [reply test/sharness/Rules.mk]: @jin-tron from what i can tell it looks like the binary file is built as part of the Makefile in the go-btfs/test/sharness/ directory.


`go build  -asmflags=all=-trimpath="/Users/richardlee/go" -gcflags=all=-trimpath="/Users/richardlee/go" -ldflags="-X "github.com/TRON-US/go-btfs".CurrentCommit=2976c69cf-dirty" -o "cmd/btfs/btfs" "github.com/TRON-US/go-btfs/cmd/btfs"`

thanks for the CR.

**Pull request merged**